napi-2.3.0/src/env.rs: cannot find trait `ToNapiValue` in this scope

Version 2.3.0 doesn't compile for me:
```
   Compiling napi v2.3.0
error[E0405]: cannot find trait `ToNapiValue` in this scope
    --> /Users/niklas/.cargo/registry/src/github.com-1ecc6299db9ec823/napi-2.3.0/src/env.rs:1043:8
     |
1043 |     V: ToNapiValue,
     |        ^^^^^^^^^^^
     |
    ::: /Users/niklas/.cargo/registry/src/github.com-1ecc6299db9ec823/napi-2.3.0/src/js_values/mod.rs:591:1
     |
591  | pub trait NapiValue: Sized + NapiRaw {
     | ------------------------------------ similarly named trait `NapiValue` defined here
     |
help: a trait with a similar name exists
     |
1043 |     V: NapiValue,
     |        ~~~~~~~~~
help: consider importing this trait
     |
1    | use crate::bindgen_prelude::ToNapiValue;
     |

For more information about this error, try `rustc --explain E0405`.
error: could not compile `napi` due to previous error
```

--- a/crates/napi/src/env.rs
+++ b/crates/napi/src/env.rs
@@ -7,7 +7,7 @@ use std::mem;
 use std::os::raw::{c_char, c_void};
 use std::ptr;
 
-#[cfg(all(feature = "tokio_rt", feature = "napi4"))]
+#[cfg(all(feature = "napi4"))]
 use crate::bindgen_runtime::ToNapiValue;
 use crate::{
   async_work::{self, AsyncWorkPromise},

